### ISSUE TITLE
Vereinsantragssachen noch erreichbar?

### ISSUE THREAD

**Issue by theZiz** [theZiz/ods-upgrade#2]
Ich muss gucken, ob die Vereinsantragssachen da sind, wo sie für potentielle Neumitgleider auffindbar sind.

**Comment by ods-homer** [comment]
habe die vereinssachen nur über hilfe gefunden, wäre schick, die etwas prominenter zu verlinken.
der link auf das antragsformular ist 404, der zu paypal funzt

**Comment by theZiz** [comment]
> der link auf das antragsformular ist 404

Das ist das Hauptproblem, was ich mal fixen sollte. :sweat_smile:

**Comment by theZiz** [comment]
So, 404 ist gefixt, nun noch die Vereinsplatzierung

**Comment by theZiz** [comment]
So, hab mal was in den Footer gepackt. Für den Header fands ich zu spärlich. Das können wir mal machen, wenn wir ne richtige, gepflegte Vereinswebseite haben und nicht nur 4 Hilfepunkte. :roll_eyes: 

![grafik](https://user-images.githubusercontent.com/385145/103172942-aefb6e00-4857-11eb-8452-8f8af2ec9c0e.png)

**Comment by ods-homer** [comment]
supi, du bist FAST fertig mit dieser baustelle 👍 
der link zu den rabatt-partnern funzt nicht ganz:
https://www.outdoorseiten.net/vb5/showthread.php?53806-Mitglied-werden-Rabatte-unserer-Partner
_Entschuldige, Du bist nicht autorisiert diese Seite zu sehen._

**Comment by ods-homer** [comment]
nachtrag: stimmt die postdresse auf dem Antrag überhaupt noch?
sieht mir sehr nach rosi aus...

**Comment by theZiz** [comment]
@ods-homer Ich habe deinen letzten Kommentar gelöscht. Bitte bedenke, github ist öffentich und Realnamen alter Vorstandsmitglieder gehen hier niemanden was an. :wink:

**Comment by theZiz** [comment]
> supi, du bist FAST fertig mit dieser baustelle +1
> der link zu den rabatt-partnern funzt nicht ganz:
> https://www.outdoorseiten.net/vb5/showthread.php?53806-Mitglied-werden-Rabatte-unserer-Partner
> _Entschuldige, Du bist nicht autorisiert diese Seite zu sehen._

Geht, wenn man eingeloggt ist. Ich füg den Hinweis mal hinzu. ;)

**Comment by ods-homer** [comment]
erhalte eingeloggt die fehlermeldung

**Comment by theZiz** [comment]
![Hmm...](https://media.giphy.com/media/UvwI1X7XkbXq0/giphy.gif)

**Comment by theZiz** [comment]
Narv, das ist nur für Vereinsmitglieder... "Hey, werde Mitglied, um zu sehen, was du bekommst!" -_-

**Comment by theZiz** [comment]
Ich leite es mal an den Vorstand weiter ;)

**Comment by ods-homer** [comment]
> 
> 
> Narv, das ist nur für Vereinsmitglieder... "Hey, werde Mitglied, um zu sehen, was du bekommst!" -_-

bin vereinsmitglied, mein testdummy nicht - beide sehen die fehlermeldung.

**Comment by theZiz** [comment]
> > Narv, das ist nur für Vereinsmitglieder... "Hey, werde Mitglied, um zu sehen, was du bekommst!" -_-
> 
> bin vereinsmitglied, mein testdummy nicht - beide sehen die fehlermeldung.

Ja, stimmt. :facepalm: Um so komischer. Zumindest für deinen Realaccount sollte es gehen.
